R/FilterStateExpr.R

```diff
@@ -109,11 +111,11 @@ FilterStateExpr <- R6::R6Class( # nolint
     },
 
     #' @description
-    #' Sets filtering state.
+    #' Does nothing. Exists for compatibility.
     #'
     #' @param state (`teal_slice`)
     #'
-    #' @return `self` invisibly
+    #' @return Returns `self`, invisibly.
```

Comment:
Why make it different from line 142 below? `` `NULL`, invisibly``

It seems to already have the "Returns" header

![image](https://github.com/insightsengineering/teal.slice/assets/211358/28a3f451-a2b5-4bcc-8ac2-2bbfded99ae0)


```suggestion
    #' @return `self`, invisibly.
```

Reply:
Good point :+1: